test

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
-set -e
-
-# Verify 'convert' (ImageMagick) is available.
```

Comment:
Can we preserve this comment?  It also isn't right away clear to me why we need to test this separately.